package.json

```diff
@@ -3,28 +3,36 @@
   "description": "Node process monitoring tool",
   "version": "0.0.14",
   "author": "Rohini Harendra <rohini.raghav@gmail.com>",
-  "os": [ "linux" ],
-  "cpu": [ "x64", "ia32" ],
+  "os": [
+    "linux"
+  ],
+  "cpu": [
+    "x64",
+    "ia32"
+  ],
   "repository": {
-      "type": "git",
-      "url": "https://github.com/yahoo/monitr.git"
+    "type": "git",
+    "url": "https://github.com/yahoo/monitr.git"
   },
   "bugs": {
-      "url" : "http://github.com/yahoo/monitr/issues"
+    "url": "http://github.com/yahoo/monitr/issues"
   },
   "keywords": [
-    "node-monitoring", "node-status"
+    "node-monitoring",
+    "node-status"
   ],
-  "licenses":[
-        {
-            "type" : "BSD",
-            "url" : "https://github.com/yahoo/monitr/blob/master/LICENSE"
-        }
-    ],
-  "engines": { "node": ">=0.10" },
+  "licenses": [
+    {
+      "type": "BSD",
+      "url": "https://github.com/yahoo/monitr/blob/master/LICENSE"
+    }
+  ],
+  "engines": {
+    "node": ">=0.10"
+  },
   "dependencies": {
     "bindings": "*",
-    "nan":"*"
+    "nan": ">=1.4.0"
```

Comment:
Hi @cloudlakecho these numbers are package version numbers. If you should be able to find a description of how these are used by running `npm help json` or `npm help 7 semver` on the commandline.